config/settings/production.py

```diff
@@ -255,9 +256,14 @@
 
 HEROKU_TOKEN = env("HEROKU_TOKEN", default=None)
 HEROKU_APP_NAME = env("HEROKU_APP_NAME", default=None)
+
 if HEROKU_TOKEN and HEROKU_APP_NAME:
     METACI_WORKER_AUTOSCALER = "metaci.build.autoscaling.HerokuAutoscaler"
 
+METACI_APPS = env("METACI_APPS", default=None)
+# Autoscalers are defined per METACI_APP
+AUTOSCALERS = json.loads(env("AUTOSCALERS", default=None))
```

Comment:
For backwards compatibility, if the environment variable isn't set, let's construct a default AUTOSCALERS dict that uses the old HEROKU_APP and HEROKU_WORKER_NAME vars and old default queues.